cubids/cubids.py

```
@@ -401,28 +497,32 @@ def change_filename(self, filepath, entities):
         if len(filename) > 0:
             filename = sub_ses + "_" + filename + "_" + suffix + old_ext
         else:
+            # XXX: Why is filename here if it's an empty string?
```

we should just raise an exception here probably

---

Good idea. I'll add that.